test/rust_src/src/windows-tests.el

```diff
@@ -189,3 +189,21 @@
 ;; frames inside tests (see https://github.com/remacs/remacs/issues/1429).
 (ert-deftest window-pixel-height-before-size-change ()
   (window-pixel-height-before-size-change))
+
+;; TODO: find a way to ignore COLUMNS, LINES. Otherwise it fails when run by hand.
```

Comment:
It looks like the columns and lines returned by `window-scroll-bars` will vary. It is calculated based on width and height of the scroll bar. For example, the test fails if I open emacs and run the test locally. 

I'm not sure the best way to make this test lest fragile.

Reply:
hmmmm

Reply:
So I believe I kind of found the reason why the column value varied. When Remacs is run without a window system (in the terminal), there is no scroll bar and columns is always 0. 

I haven't pinned down the exact line where this happens, but it seems to be how it works.

To work around it - I check [window-system](https://www.gnu.org/software/emacs/manual/html_node/elisp/Window-Systems.html) in the test.